itests/eudico_hc_test.go

```diff
@@ -783,36 +765,19 @@ func (ts *eudicoSubnetSuite) testDataRaces(t *testing.T) {
 		}
 	}()
 
-	go func() {
```

Comment:
Why did you decide to go from 3 nodes to 2 nodes in this test? (I guess that for readibility?)

Reply:
Those 2 nodes are nodes for the rootnet and the primary subnet. I beleive there is no much sense to start 3 nodes or even 2.
So the setup is as follows:
- 2 nodes in the rootnet with dummy consensus
- 2 nodes in the subnet with Mir
- 4 side subnets with 1 node in each with PoW, Dummy, Delegated, PoW to provide concurrent access to subnet APIs